cicd/index.adoc

```diff
@@ -15,19 +15,19 @@ toc::[]
 
 [id="openshift-builds"]
 == OpenShift Builds
-OpenShift Builds provides you the following option to configure and run a build:
+OpenShift Builds provides you the following options to configure and run a build:
 
-// * Builds using Shipwright
-// +
-// An extensible build framework based on the Shipwright project, which you can use to build container images on an {product-title} cluster. You can build container images from source code and Dockerfile by using image build tools, such as Source-to-Image (S2I) and Buildah.
-// +
-// For more information, see link:https://docs.openshift.com/builds/latest/about/overview-openshift-builds.html[Overview of Builds].
+* Builds using Shipwright
++
+An extensible build framework based on the Shipwright project, which you can use to build container images on an {product-title} cluster. You can build container images from source code and Dockerfile by using image build tools, such as Source-to-Image (S2I) and Buildah.
```

Comment:
This is just a suggestion but you could rephrase the bullet like:

```suggestion
Builds using Shipwright is an extensible build framework based on the Shipwright project. You can use it to build container images on an {product-title} cluster. You can also build container images from source code and Dockerfile by using image build tools, such as Source-to-Image (S2I) and Buildah.
+
For more information, see xref.....
```
Not sure about adding the "also" I'll let you be the judge of that.

Reply:
Fixed. `also` not required in this context.